spring-integration-sftp/src/main/java/org/springframework/integration/sftp/session/DefaultSftpSessionFactory.java

```diff
@@ -43,11 +47,22 @@
  * @author Gunnar Hillert
  * @author Gary Russell
  * @author David Liu
+ * @author Pat Turner
  *
  * @since 2.0
  */
 public class DefaultSftpSessionFactory implements SessionFactory<LsEntry>, SharedSessionCapable {
 
+	private static final Log logger = LogFactory.getLog(DefaultSftpSessionFactory.class);
+
+	private final ReentrantReadWriteLock sharedSessionLock = new ReentrantReadWriteLock();
+
+	private final UserInfoWrapper userinfoWrapper = new UserInfoWrapper();
```

Comment:
Minor, but the code should be ideal :smile: 
Interfaces for properties and the name for the last one must be fixed as well - user**I**nfoWrapper